var/spack/repos/builtin/packages/elemental/package.py

```diff
@@ -0,0 +1,131 @@
+    variant('scalapack', default=False,
+            description='Build with ScaLAPACK library')
+
+    depends_on('cmake', type='build')
+    # Note that this forces us to use OpenBLAS until #1712 is fixed
+    depends_on('blas', when='~openmp_blas ~int64_blas') # Hack until issue #1712 is fixed
+    # Hack to forward variant to openblas package
+    # Allow Elemental to build internally when using 8-byte ints
+    depends_on('openblas +openmp', when='+openmp_blas ~int64_blas')
+    # Note that this forces us to use OpenBLAS until #1712 is fixed
+    depends_on('lapack', when='~openmp_blas')
+    depends_on('metis')
+    depends_on('metis +int64', when='+int64')
+    depends_on('mpi')
+    # Allow Elemental to build internally when using 8-byte ints
+    depends_on('scalapack', when='+scalapack ~int64_blas')
+    depends_on('python', when='+python')
+
+    def build_type(self):
+        """Returns the correct value for the ``CMAKE_BUILD_TYPE`` variable
+        :return: value for ``CMAKE_BUILD_TYPE``
+        """
+        if '+debug' in self.spec:
+            return 'Debug'
+        else:
+            return 'Release'
+
+    def cmake_args(self):
+        args = ['-DCMAKE_INSTALL_MESSAGE:STRING=LAZY',
+                '-DEL_PREFER_OPENBLAS:BOOL=TRUE',
+                '-DEL_DISABLE_SCALAPACK:BOOL={0}'.format((
+                    'OFF' if '+scalapack' in self.spec else 'ON')),
+                '-DGFORTRAN_LIB=libgfortran.so',
+                '-DBUILD_SHARED_LIBS:BOOL={0}'.format((
+                    'ON' if '+shared' in self.spec else 'OFF')),
+                '-DEL_HYBRID:BOOL={0}'.format((
+                    'ON' if '+hybrid' in self.spec else 'OFF')),
+                '-DEL_C_INTERFACE:BOOL={0}'.format((
+                    'ON' if '+c' in self.spec else 'OFF')),
+                '-DINSTALL_PYTHON_PACKAGE:BOOL={0}'.format((
+                    'ON' if '+python' in self.spec else 'OFF')),
+                '-DEL_DISABLE_PARMETIS:BOOL={0}'.format((
+                    'OFF' if '+parmetis' in self.spec else 'ON')),
+                '-DEL_DISABLE_QUAD:BOOL={0}'.format((
+                    'OFF' if '+quad' in self.spec else 'ON')),
+                '-DEL_USE_64BIT_INTS:BOOL={0}'.format((
+                    'ON' if '+int64' in self.spec else 'OFF')),
+                '-DEL_USE_64BIT_BLAS_INTS:BOOL={0}'.format((
+                    'ON' if '+int64_blas' in self.spec else 'OFF'))]
+        if '+int64_blas' in self.spec:
+            args.extend(['-DEL_BLAS_SUFFIX:STRING={0}'.format((
+                '_64_' if '+int64_blas' in self.spec else '_')),
+                         '-DCUSTOM_BLAS_SUFFIX:BOOL=TRUE']),
+            if '+scalapack' in self.spec:
+                args.extend(['-DEL_LAPACK_SUFFIX:STRING={0}'.format((
+                    '_64_' if '+int64_blas' in self.spec else '_')),
+                             '-DCUSTOM_LAPACK_SUFFIX:BOOL=TRUE']),
+        else:
+            lapack = self.spec['lapack'].lapack_libs
```

Comment:
you never use them alone, why not haveing `lapack_blas =  self.spec['lapack'].lapack_libs+self.spec['lapack'].blas_libs`?